`NullPointerException` when handling embeds

There is a regression that suddenly you get a `NullPointerException` when it worked perfectly well before to give `null` as embed to `sendMessage` which meant to not send an embed.

```
Caused by: java.lang.NullPointerException
	at org.javacord.core.entity.message.MessageBuilderBaseDelegateImpl.prepareEmbeds(MessageBuilderBaseDelegateImpl.java:709) ~[main/:?]
	at org.javacord.core.entity.message.MessageBuilderBaseDelegateImpl.send(MessageBuilderBaseDelegateImpl.java:464) ~[main/:?]
	at java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1106) [?:?]
	at java.util.concurrent.CompletableFuture.thenCompose(CompletableFuture.java:2235) [?:?]
	at org.javacord.core.entity.message.MessageBuilderBaseDelegateImpl.send(MessageBuilderBaseDelegateImpl.java:446) ~[main/:?]
	at org.javacord.api.entity.message.MessageBuilder.send(MessageBuilder.java:117) ~[main/:?]
	at org.javacord.api.entity.message.Messageable.sendMessage(Messageable.java:348) ~[main/:?]
	at org.javacord.api.entity.message.Messageable.sendMessage(Messageable.java:330) ~[main/:?]
	at net.kautler.discord.bot.translator.TranslatorDiscordBot.sendReply(TranslatorDiscordBot.java:539) ~[main/:?]
	... 16 more
```

--- a/javacord-core/src/main/java/org/javacord/core/entity/message/MessageBuilderBaseDelegateImpl.java
+++ b/javacord-core/src/main/java/org/javacord/core/entity/message/MessageBuilderBaseDelegateImpl.java
@@ -245,8 +245,7 @@ public class MessageBuilderBaseDelegateImpl implements MessageBuilderBaseDelegat
 
     @Override
     public void addEmbeds(List<EmbedBuilder> embeds) {
-        this.embeds.addAll(embeds);
-        embedsChanged = true;
+        embeds.forEach(this::addEmbed);
     }
 
     @Override

